editor/animation/animation_track_editor.cpp

```diff
@@ -8268,6 +8336,11 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	track_copy_select->set_hide_root(true);
 	track_copy_vbox->add_child(track_copy_select);
 	track_copy_dialog->connect(SceneStringName(confirmed), callable_mp(this, &AnimationTrackEditor::_edit_menu_pressed).bind(EDIT_COPY_TRACKS_CONFIRM));
+
+	read_only_dialog = memnew(AcceptDialog);
```

Comment:
You could create the dialog on demand in `popup_read_only_dialog()`, it will rarely be needed.

Reply:
I think it's okay to generate it here for consistency with other dialogs.